### PR TITLE
Updated training scripts

### PR DESCRIPTION
**Pull request by djgagne**
All training scripts now keras 3 and use new models. Removed direct tensorflow calls where I could find them.

### CODE REVIEW

**Review by charlie-becker — APPROVED** [review]
Thanks for the update. It would be nice to add an additional  test / assert for the MC dropout method. Would also be good to test the train script, but that may need to come later (with config changes).
